src/components/ModalContent/ModalContent.js

```diff
@@ -18,16 +18,27 @@ import { stripesConnect } from '@folio/stripes/core';
 
 import { getOpenRequestsPath } from '../util';
 
-import { loanActionMutators, refundClaimReturned, loanActions } from '../../constants';
+import {
+  loanActionMutators,
+  refundClaimReturned,
+  loanActions,
+  CIRCULATION_BFF_LOANS_INTERFACE_NAME,
+  CIRCULATION_BFF_LOANS_INTERFACE_VERSION,
+  CIRCULATION_BFF_LOANS_INTERFACE_ERROR,
+} from '../../constants';
 
 import css from './ModalContent.css';
 
 export const getMutatorFunction = (stripes, mutator, loanAction) => {
-  if (stripes?.config?.enableEcsRequests && loanAction === loanActionMutators.DECLARE_LOST) {
-    if (stripes.hasInterface('circulation-bff-loans', '1.4')) {
+  if (stripes?.config?.enableEcsRequests && (loanAction === loanActionMutators.DECLARE_LOST || loanAction === loanActionMutators.CLAIMED_RETURNED)) {
+    const isCorrectCirculationBFFLoansInterface = stripes.hasInterface(CIRCULATION_BFF_LOANS_INTERFACE_NAME, CIRCULATION_BFF_LOANS_INTERFACE_VERSION);
+
+    if (isCorrectCirculationBFFLoansInterface && loanAction === loanActionMutators.DECLARE_LOST) {
```

Comment:
move loanAction === loanActionMutators.DECLARE_LOST to a variable, I see you used it twice already

Reply:
Fixed